### ISSUE TITLE
Single Cell RNA-seq reveals ectopic and aberrant lung resident cell populations in Idiopathic Pulmonary Fibrosis

### ISSUE THREAD

**Issue by ipediez** [ebi-ait/hca-ebi-wrangler-central#750]
### Project short name:
lungPopulationsIPF

### Primary Wrangler:
All wranglers. 
Metadata lead: Irene
Data lead: Enrique

### Associated files

* Google Drive: https://drive.google.com/drive/folders/1rYux17a7JCPbnajxGaGlAIylTKkrG2qm?usp=sharing
* Ingest: https://contribute.data.humancellatlas.org/projects/detail?uuid=c16a754f-5da3-46ed-8c1e-6426af2ef625

### Published study links

* Paper: https://www.science.org/doi/10.1126/sciadv.aba1983

* Accessioned data: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831

### Key Events

- [x] Convert published metadata to HCA spreadsheet
- [x] Manually curate dataset to meet HCA metadata standard
- [x] Collect any matrix and cell-type annotation files
- [x] Upload sheet to validate metadata
- [x] Check linking using ingest graph validator
- [x] Transfer raw files to ingest to validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [x] Submit dataset to Production 
- [ ] Complete the Export SOP
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by ipediez** [comment]
We started wrangling this dataset as part of the "Wrangler workshop - remove obstacles in the flow"

- @ipediez to finish wrangling the metadata and inform the secondary wrangler about which fields/tabs were not reviewed during the workshop
- @ESapenaVentura to wrangle the available data
- @idazucchi or @Wkt8 to secondary wrangle the information that wasn't filled in during the workshop

**Comment by ESapenaVentura** [comment]
hca-util area: eaf44850-d35f-4ab0-a53e-f81c5f236289

**Comment by ipediez** [comment]
Fields waiting on secondary review:

- [ ] Enrichment protocol tab

**Comment by gabsie** [comment]
@Wkt8 to secondary review this.

**Comment by Wkt8** [comment]
Looks great to me! Fixed a small typo with contributor names in Ingest. Apart from that, it looks great!

**Comment by MightyAx** [comment]
R16 ReExport Successful

**Comment by ESapenaVentura** [comment]
@idazucchi to look today
@ESapenaVentura: LGTM

**Comment by arschat** [comment]
After Dave's request in #1270 I am re-opening the current ticket to add fastq files that are now available in [GSE136831](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831). 

hca-util upload area already contains the analysis files, and fastqs have been uploaded there as well.
Discussing with @ESapenaVentura how to proceed with the update, while keeping the same uuids for all other entities.

I downloaded the spreadsheet from [ingest's](https://contribute.data.humancellatlas.org/projects/detail?uuid=c16a754f-5da3-46ed-8c1e-6426af2ef625) submission, and added the Sequence file tab, and wrangled them. I filled the uuids for the references of the seq file tab as well (cell suspension, library preparation protocol, sequencing protocol).

spreadsheet \w seq files here: [1da128e6-c525-42c0-be81-384998f93781_20240620-155943.xlsx](https://github.com/user-attachments/files/15954219/1da128e6-c525-42c0-be81-384998f93781_20240620-155943.xlsx)
